netstandard/ref/System.Drawing.cs

```diff
@@ -172,17 +174,19 @@ public partial struct Color
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public partial struct Point
+    public partial struct Point : System.IEquatable<System.Drawing.Point>
     {
         public static readonly System.Drawing.Point Empty;
         public Point(System.Drawing.Size sz) { throw null;}
         public Point(int dw) { throw null;}
         public Point(int x, int y) { throw null;}
+        [System.ComponentModel.BrowsableAttribute(false)]
```

Comment:
Why are we hiding IsEmpty ?

Reply:
No idea, I just work here but this is how [it came from .NET Framework](https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/Point.cs,83).

Reply:
Follow-up: `[Browsable]` is for the property grid (don't confuse it with `[EditorBrowsable]` which applies to the editor/IntelliSense). I suspect the hid from the property grid for WinForms (as a read-only property doesn't add much here).

Reply:
I suggest we leave it as-is.